doc/source/whatsnew/v0.21.0.txt

```diff
@@ -60,6 +60,7 @@ Other API Changes
 ^^^^^^^^^^^^^^^^^
 
 - Moved definition of ``MergeError`` to the ``pandas.errors`` module.
+- :func:`eval` will now raise a ``ValueError`` when item assignment malfunctions, or inplace operations are specified, but there is no item assignment in the expression (:issue:`16732`)
```

Comment:
Added it to the breaking section, since technically, passing in `inplace=True` without item assignment is disallowed now.